src/helpers/graphql.js

```diff
@@ -77,7 +77,10 @@ query{
 `;
 
 export function queryAllTopics() {
-  return client.query({ query: ALL_TOPICS }).then((res) => {
+  return client.query({
+    query: ALL_TOPICS,
+    fetchPolicy: 'network-only',
```

Comment:
Ya I am not sure actually if we need this for the topics and oracles. But I was stuck on why the syncInfo for was not changing. I had to put `network-only` to force apollo to actually call `querySyncInfo`. Without it, it would just use the cached sync info. Stuck on this for a while lol...